tests/deci_core_unit_test_suite_runner.py

```diff
@@ -85,6 +86,7 @@ def _add_modules_to_unit_tests_suite(self):
         self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(IoULossTest))
         self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(TestDetectionDatasetSubsampling))
         self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(TestDetectionDatasetSubclassing))
+        self.unit_tests_suite.addTest(self.test_loader.loadTestsFromModule(QuantizationUtilityTest))
```

Comment:
Lets check the dependencies for pytorch-quantization + licenses, if there are no req for GPUs, then lets add it to our requirements and run the unit tests in the CI.